[Doc Improvement] Update TeamsJS terminology as "library" (from "SDK") (#7306)
MicrosoftDocs/msteams-docs: msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

> [!TIP]
> Get participant IDs and tenant IDs from the [tab SSO authentication](../tabs/how-to/authentication/tab-sso-overview.md).

> The `Meeting` API must have `meetingId`, `participantId`, and `tenantId` as URL parameters. The parameters are available as part of the TeamsJS library and bot activity.

Reviewer: ```suggestion
The `Meeting` API must have `meetingId`, `participantId`, and `tenantId` as URL parameters. The parameters are available as part of the TeamsJS library and bot activity.
```

The following table includes the query parameters:

|Value|Type|Required|Description|
|---|---|----|---|
|**meetingId**| String | Yes | The meeting identifier is available through Bot Invoke and TeamsJS library.|

> |**participantId**| String | Yes | The participant ID is the user ID. It's available in Tab SSO, Bot Invoke, and Teams JS library. It's recommended to get a participant ID from the Tab SSO. |

Reviewer: ```suggestion
|**participantId**| String | Yes | The participant ID is the user ID. It's available in Tab SSO, Bot Invoke, and TeamsJS library. It's recommended to get a participant ID from the Tab SSO. |
```

>
> * When an in-meeting notification is invoked, the content is presented as a chat message.
> * Currently, sending targeted notifications and support for webapp are not supported.

> > * You must invoke the [submitTask()](../task-modules-and-cards/task-modules/task-modules-bots.md#submit-the-result-of-a-task-module) function to dismiss automatically after a user takes an action in the web view. This is a requirement for app submission. For more information, see TeamsJS library [task module](/javascript/api/@microsoft/teams-js/microsoftteams.tasks?view=msteams-client-js-latest#submittask-string---object--string---string---&preserve-view=true).

Reviewer: ```suggestion
> * You must invoke the [submitTask()](../task-modules-and-cards/task-modules/task-modules-bots.md#submit-the-result-of-a-task-module) function to dismiss automatically after a user takes an action in the web view. This is a requirement for app submission. For more information, see [TeamsJS library task module](/javascript/api/@microsoft/teams-js/microsoftteams.tasks?view=msteams-client-js-latest#submittask-string---object--string---string---&preserve-view=true).
```

Author: I intentionally left "TeamsJS library" out of the link text, as this is a non-standard way of linking to individual API reference topics.
